frappe/core/page/background_jobs/background_jobs_outer.html

```diff
@@ -2,9 +2,12 @@
 	<p>
 		<div class="checkbox">
 			<label>
-			<input type="checkbox" class="show-failed"> {{ __("Show failed jobs") }}
+				<input type="checkbox" class="show-failed"> {{ __("Show failed jobs") }}
 			</label>
 		</div>
+		<div>
+			<button type="button" class="btn btn-primary remove-failed"> {{ __("Remove failed jobs") }}</button>
```

Comment:
Can we show this only when necessary?

Reply:
Added a fix for that, it'll only show up when there are jobs in the queue(s)